src/views/virtualmachines/KubevirtPluginContext.ts

```diff
@@ -0,0 +1,15 @@
+import { createContext } from 'react';
+
+import * as kubevirtDynamicPluginSDK from '@openshift-console/dynamic-plugin-sdk';
+
+type KubevirtPluginData = {
+  kubevirtSDK: typeof kubevirtDynamicPluginSDK;
```

Comment:
This is the `openshift-console/dynamic-plugin-sdk` it's not specific to kubevirt. Maybe rename it to `dynamicPluginSDK`